servers/zts/src/main/java/com/yahoo/athenz/zts/ZTSImpl.java

```diff
@@ -2678,7 +2696,7 @@ public Response postInstanceRegisterInformation(ResourceContext ctx, InstanceReg
             // able to validate the certificate during refresh operations
 
             if (insertX509CertRecord(ctx, cn, provider, certReqInstanceId, certSerial,
-                    InstanceProvider.ZTS_CERT_USAGE_CLIENT.equalsIgnoreCase(certUsage)) == null) {
+                    InstanceProvider.ZTS_CERT_USAGE_CLIENT.equalsIgnoreCase(certUsage), null, null) == null) {
```

Comment:
Currently, always put null (the columns are nullable). Will insert the real values once hca starts sending them as part of the request.

Reply:
expiryTime is already available - it must come from the certificate that ZTS generates. only hostname is supposed to come from the service identity agent. We already get those when the host is requesting the hostname to be included in the car, so we there is no need to wait for any changes in the agent.

Reply:
Fixed.